Library/Homebrew/os/mac/version.rb

```diff
@@ -15,7 +15,6 @@ class Version < ::Version
         snow_leopard:  "10.6",
         leopard_64:    "10.5",
         leopard:       "10.5",
-        tiger:         "10.4",
```

Comment:
This broke all casks referencing `:tiger`.

Reply:
@reitermarkus Homebrew/brew has never run on Tiger. Is there a `brew readall` equivalent for Homebrew/cask that we could add to CI to ensure that we don't break Cask syntax? Is there an open PR working on this or are you asking me to create one?

Reply:
@reitermarkus Note that support for pre-Mavericks is going to be dropped in 2.0.0 so it would be good if @homebrew/cask could work on some sort of tooling to make sure this doesn't happen again. For example I was notified through a CI failure and fixed Linuxbrew/homebrew-core issues in this PR.

Reply:
> For example I was notified through a CI failure and fixed Linuxbrew/homebrew-core issues in this PR.

Thanks, Mike. I appreciate that.